pkg/registry/core/pod/storage/eviction.go

```diff
@@ -307,12 +313,25 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 	return &metav1.Status{Status: metav1.StatusSuccess}, nil
 }
 
-func addConditionAndDeletePod(r *EvictionREST, ctx context.Context, name string, validation rest.ValidateObjectFunc, options *metav1.DeleteOptions) error {
-	if feature.DefaultFeatureGate.Enabled(features.PodDisruptionConditions) {
-		pod, err := getPod(r, ctx, name)
-		if err != nil {
-			return err
+func addConditionAndDeletePod(r *EvictionREST, ctx context.Context, pod *api.Pod, deleteOptions *metav1.DeleteOptions) error {
```

Comment:
revert the signature change here, we don't need the looked-up pod any more

Reply:
reverted the name back to options for a smaller diff. Pod up for a discussion in https://github.com/kubernetes/kubernetes/pull/116554#discussion_r1139283729